File: app/src/main/java/com/babestudios/companieshouse/data/model/Matches.java
package com.babestudios.companieshouse.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Matches {

	@SerializedName("title")
	public List<Integer> title = new ArrayList<>();

	@SerializedName("snippet")
	public List<Object> snippet = new ArrayList<>();

}

File: data/src/main/java/com/babestudios/companyinfo/data/model/search/Matches.kt
package com.babestudios.companyinfo.data.model.search

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

import java.util.ArrayList

@Parcelize
class Matches (

	@SerializedName("title")
	var title: List<Int> = ArrayList(),

	@SerializedName("snippet")
	var snippet: List<String> = ArrayList()

): Parcelable
